### PR TITLE
Align logo usage

### PR DESCRIPTION
**Pull request by egekorkan**
replicating https://github.com/eclipse-thingweb/.github/pull/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-<img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width="300">
-
-# Thing Description Tools
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/main/brand/logos/td-tools_for_dark_bg.svg">
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/td-tools.svg">
+    <img title="Eclipse Thingweb TD Tools" alt="Eclipse Thingweb logo with TD Tools" src="https://github.com/eclipse-thingweb/thingweb/raw/main/brand/logos/td-tools.svg" width="300">
```
Comment (danielpeintner) [inline README.md:5]:
```suggestion
    <img title="Eclipse Thingweb TD Tools" alt="Eclipse Thingweb logo for TD Tools" src="https://github.com/eclipse-thingweb/thingweb/raw/main/brand/logos/td-tools.svg" width="300">
```

Comment (egekorkan) [reply README.md:5]: I have actually used the same alt text in all PRs to give a meaning like: We use the Eclipse Thingweb logo and then add the "component"

Comment (danielpeintner) [reply README.md:5]: I don't have a strong opinion either way

Comment (relu91) [reply README.md:5]: Me neither I think we can go with "with" to make it quicker.

**Pull request merged**